### PR TITLE
Vss code refactor and add frames sizes in config files

### PR DESCRIPTION
**Pull request by MalwinaKozak**

### CODE REVIEW

File: code/serial/Redis.py [hunk 1]
```diff
@@ -62,8 +70,12 @@ def start_redis(self, queue):
             
             # get data bytes and crc8
             data_bytes = bytearray()
-            for _ in range(frame_size.get(id, 2)+1):
-                data_bytes.extend([queue.get()])
+            if is_ecumaster_frame:
+                for _ in range(self.can_parser.get_ecumaster_frame_size(id & 0x7ff)):
```
Comment (cepekLP) [inline code/serial/Redis.py:74]: add frame size to variable
eg size = self.can_parser.get_ecumaster_frame_size(id & 0x7ff)

File: code/serial/Redis.py [hunk 2]
```diff
@@ -109,7 +121,7 @@ def init_filters(self):
         self.ecumaster = Ecumaster()
         self.brakes = Brakes()
         self.gps = GPS()
-        self.abs = Abs()
+        self.abs = Vss()
```
Comment (cepekLP) [inline code/serial/Redis.py:124]:
```suggestion
        self.vss = Vss()
```

File: code/sensors/VSS.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            self.acc_time += 0.1
+            self.acc_distance += (dataframe["front_left_speed"] + dataframe["front_right_speed"])/2 / 3.6 * 0.1
+        elif self.acc_distance >= self.acc_max_distance:
+                self.acc_time = 0
+                self.acc_distance = 0
+
+        if self.acc_time !=0:
+            dataframe["acc_time"] = self.acc_time
+
+    def callback_front(self, dataframe):
```
Comment (cepekLP) [inline code/sensors/VSS.py:45]: remove

**Review by cepekLP — CHANGES_REQUESTED** [review]

File: code/serial/Redis.py [hunk 4]
```diff
@@ -45,14 +45,22 @@ def start_redis(self, queue):
         self.can_parser.register_telemetry_callback('0x50b', self.brakes.callback)
         self.can_parser.register_telemetry_callback('0x509', self.gps.callback)
         self.can_parser.register_ecumaster_callback('0x602', self.gps.callback_additional)
-        self.can_parser.register_telemetry_callback('0x503', self.abs.callback)
+        self.can_parser.register_telemetry_callback('0x503', self.vss.callback)
```
[the remark this answers is not in the document]
Comment (MalwinaKozak) [reply code/serial/Redis.py]: callback deleted

**Review by cepekLP — APPROVED** [review]
🌵

**Pull request merged**